src/lib/steps.ts

```diff
@@ -60,13 +60,6 @@ export type FillnaStep = {
   value: PrimitiveType;
 };
 
-// export type FilterStep = {
```

Comment:
Not very important but if would great if you could directly update the first commit to remove these commented lines.